source/core_sntp_serializer.c

```diff
@@ -242,7 +257,7 @@ static int64_t safeTimeDifference( uint32_t serverTimeSec,
      *, we will return the maximum value representable by signed 2^31, i.e. 2147483647, resulting in
      * an inaccuracy of 1 second in the clock-offset value.
      */
-    if( diffWithNoEraAdjustment == INT32_MIN )
+    if( absoluteOf( diffWithNoEraAdjustment ) == CLOCK_OFFSET_MAX_TIME_DIFFERENCE )
```

Comment:
There is a variable `absSameEraDiff` for the same expression. Does it make sense to move that variable above and use here?